src/sections/Community/Web-based-from/index.js

```diff
@@ -7,6 +7,38 @@ import { Field, Formik, Form } from "formik";
 import axios from "axios";
 import { Link } from "gatsby";
 
+const validateEmail = (value) => {
+  let error;
+
+  if (!value) {
+    error = "Required";
+  } else if (!/^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}$/.test(value)) {
```

Comment:
Is this the same regex used by the other site forms?

Reply:
Yes.